libs/kdb/kdb-cmn.c

```diff
@@ -444,6 +928,27 @@ rc_t KDBMakeSubPath ( struct KDirectory const *dir,
 }
 
 
+/* KDBIsPathUri
+ * A hack to get some of VFS into KDB that is too tightly bound to KFS
+ */
+
+bool KDBIsPathUri (const char * path)
```

Comment:
Can you remove this function? It got removed in engineering a while ago, but looks like it snuck back in?

Reply:
done.